.github/workflows/build-test-n-publish.yml

```diff
@@ -19,7 +19,7 @@ on:
       **
   pull_request:
     paths-ignore:  # NOTE: cannot be combined with "paths"
-    - docs/**
+    - docs/**.rst
```

Comment:
This workaround isn't exactly correct, though. The problem isn't with the ignore spec but rather with the fact that branch protection doesn't work well with multiple workflows. When I have time, I'll try to merge them into one using a better dynamic skipping mechanism, like the one pip repo uses.